core/src/main/java/tc/oc/pgm/rage/RageModule.java

```diff
@@ -23,15 +30,23 @@ public MatchModule createMatchModule(Match match) {
 
   @Override
   public Collection<MapTag> getTags() {
-    return TAGS;
+    return tags;
   }
 
   public static class Factory implements MapModuleFactory<RageModule> {
     @Override
     public RageModule parse(MapFactory factory, Logger logger, Document doc)
         throws InvalidXMLException {
+      Set<MapTag> tags = new HashSet<>();
       if (doc.getRootElement().getChild("rage") != null) {
-        return new RageModule();
+        // If the blitz module is loaded with Rage, display "Blitz: Rage" on scoreboard instead of
+        // "Blitz and Rage"
+        if (doc.getRootElement().getChild("blitz") != null) {
```

Comment:
yeah... both modules relying on eachother sounds like a circular dependency, and that's going to be annoying.

Also, you cannot just check for element because this **can** fail.

A map with the following will match this check, but will not actually have blitz enabled:
```
<blitz>
  <lives>oo</lives>
</blitz>
```